src/test/java/com/rultor/agents/daemons/StartsDaemonITCase.java

```diff
@@ -70,6 +70,11 @@
 public final class StartsDaemonITCase {
 
     /**
+     * Rultor repo.
+     */
+    private static final String RULTOR = "yegor256/rultor";
```

Comment:
@original-brownbear `deprecatesDefaultImage` is not a legal name for local variable if Qulice is used